tests/cframework/tests.h

```diff
@@ -261,6 +261,52 @@ int init (int argc, char ** argv);
 		ELEKTRA_DIAG_RESTORE                                                                                                       \
 	}
 
+#define succeed_if_keyset_contains_key_with_value(pks, pkn, pv)                                                                            \
```

Comment:
```suggestion
#define succeed_if_keyset_contains_key_with_string(pks, pkn, pv)                                                                            \
```

Looks like this macro is only intended for keys with string values.